taskvine/src/manager/vine_worker_info.h

```diff
@@ -47,6 +47,7 @@ struct vine_worker_info {
 	int  transfer_port_active;
 
 	/* Worker condition that may affect task start or cancellation. */
+	int  worker_init;                       // if 1, worker does not accept tasks. 
```

Comment:
Together `worker_init` and  `draining` are likely to be confusing to a future reader. Instead, combine these into a single state variable with names something like `INITIALIZING, WORKING, DRAINING` and that makes it much clearer that a worker has a lifecycle in which it may or may not accept tasks.